src/Pages/contactUs/components/sendEmail.js

```diff
@@ -0,0 +1,9 @@
+
+
+export function SendEmail() {
+    return (
+        <div className="EmailSuccess">
+            <h3>Email Sent</h3>
```

Comment:
I don't think we need a new file for a div and heading surely this can be a state in the contactUs.js file

Reply:
it genuinely easier for it to be there